### PR TITLE
Add placeholder generation method to dbClient

### PR DESCRIPTION
**Pull request by dylanmacdonald**
Adds a public method for converting arbitrary structs into valid postgresql insert placeholders

### CODE REVIEW

File: dbclient/placeholders.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
+package dbclient
+
+import (
+	"reflect"
+	"strconv"
+	"strings"
+)
+
+const SQL_MAX_PLACEHOLDERS = 65535
```
Comment (mbtestfeed) [inline dbclient/placeholders.go:9]: Where does this number come from?

File: dbclient/placeholders_test.go [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+				args: [][]interface{}{
+					{struct{ Foo string }{"a"}, struct{ Foo string }{"A"}},
+					{struct{ Foo string }{"b"}, struct{ Foo string }{"B"}},
+					{struct{ Foo string }{"c"}, struct{ Foo string }{"C"}},
+				},
+			},
+			want: "($1,$2),($3,$4),($5,$6)",
+		},
+	}
+	for _, tt := range tests {
```
Comment (mbtestfeed) [inline dbclient/placeholders_test.go:50]: Seems like empty array cases are missing tests.

File: dbclient/placeholders_test.go [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+					struct{ Foo, baz, Bar string }{"c", "e", "C"},
+				},
+			},
+			want: [][]interface{}{
+				{"a", "A"},
+				{"b", "B"},
+				{"c", "C"},
+			},
+		},
+	}
```
Comment (mbtestfeed) [inline dbclient/placeholders_test.go:139]: Again, missing no-length slices from tests.

**Review by EugenRakhimov — APPROVED** [review]

**Review by mbtestfeed — APPROVED** [review]
Mostly seems fine. A couple of empty-slice test cases missing, and a comment needed on where we get that constant from, but none of these are hard blockers.

**Review by jeremy-homes — APPROVED** [review]
Looks all G, is there no lib that would do this?

**Pull request merged**